index.html

```diff
@@ -0,0 +1,1260 @@
+  <link rel='stylesheet prefetch' href='https://use.fontawesome.com/releases/v5.0.10/css/all.css'>
+  <link rel="stylesheet prefetch" href="css/styles.css">
+</head>
+
+<body>
+  <div class="hero hero-bg is-fullheight">
+    <header class="hero-head">
+      <div class="navbar">
+        <div class="container">
+          <div class="navbar-brand">
+            <a href="#" class="navbar-item logo-link">
+              <img src="https://www.epam.com/content/dam/epam/global/logo_white-blue.svg" alt="Epam logo">
+            </a>
+            <button class="navbar-burger burger" data-target="navbarMenu">
+              <span></span>
+              <span></span>
+              <span></span>
+            </button>
+          </div>
+
+          <a href="#about" class="visually-hidden">Skip to main content</a>
+
+          <nav id="navbarMenu" class="navbar-menu">
+            <div class="navbar-end">
+              <span class="navbar-item">
+                <a href="#" class="button is-white is-outlined">
+                  <span class="icon">
+                    <i class="fa fa-home"></i>
+                  </span>
+                  <span>Home</span>
+                </a>
+              </span>
+              <span class="navbar-item">
+                <a href="#" class="button is-white is-outlined">
+                  <span class="icon">
+                    <i class="fab fa-superpowers"></i>
+                  </span>
+                  <span>Examples</span>
+                </a>
+              </span>
+              <span class="navbar-item">
+                <a href="#" class="button is-white is-outlined">
+                  <span class="icon">
+                    <i class="fa fa-book"></i>
+                  </span>
+                  <span>Documentation</span>
+                </a>
+              </span>
+              <span class="navbar-item">
+                <a href="#" class="button is-white is-outlined">
+                  <span class="icon">
+                    <i class="fab fa-github"></i>
+                  </span>
+                  <span>View Source</span>
+                </a>
+              </span>
+            </div>
+          </nav>
+        </div>
+      </div>
+    </header>
+
+    <div class="hero-body" id="hero">
+      <div class="container has-text-centered">
+        <div class="column is-6 is-offset-3">
+          <h3 class="title has-text-white">We have web accessibility in mind</h3>
+          <div class="subtitle has-text-white">
+            Would you like to get the latest news?
+          </div>
+          <div class="box">
+            <form class="field is-grouped">
+              <p class="control is-expanded">
+                <input class="input">
+              </p>
+              <p class="control">
+                <button class="button is-info">Notify Me</button>
+              </p>
+            </form>
+          </div>
+        </div>
+      </div>
+    </div>
+  </div>
+
```

Comment:
I suggest adding **bread crumbs** block in this place with your styles. Something like that:

```suggestion
<div id="breadcrumbs">
        <nav class="breadcrumb" role="breadcrumb">
              <li><a href="./">MainPage</a><span class="cr">»</span></li>
              <li><a href="index.html#nav">Pictures</a><span class="cr">»</span></li>
              <li><a href="index.html#Services" class="current" aria-current="page">Services</a></li>
    </nav>
 </div>
```